app/javascript/components/ErrorBoundary.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect } from 'react'
+import {
+  ErrorBoundary as ReactErrorBoundary,
+  FallbackProps,
+  useErrorHandler as useReactErrorHandler,
+} from 'react-error-boundary'
+import { APIError } from './types'
+
+const ERROR_MESSAGE_TIMEOUT = 500
```

Comment:
Maybe make it explicit that the timeout is in milliseconds?

```suggestion
const ERROR_MESSAGE_TIMEOUT_IN_MS = 500
```